qa/tasks/cephadm.py

```diff
@@ -305,6 +305,7 @@ def ceph_bootstrap(ctx, config):
     ctx.cluster.run(args=[
         'sudo', 'chmod', '777', '/etc/ceph',
         ]);
+    add_mirror_to_cluster(ctx, config.get('docker_registry_mirror', 'vossi04.front.sepia.ceph.com:5000'))
```

Comment:
can the vossi04* be declared as a constant at the beginning of the file

Reply:
this constant belongs to sepia, not into ceph/qa. but I don't know how.

Reply:
I mean that should not be like a hard coded parameter.

Reply:
i see. it's like a magic number.

Reply:
and it's actually **wrong** for other teuthology deployments.

Reply:
The customized values should be controlled by yaml files in a suite. You need to raise an exception or just ignore adding registry if corresponding key is missing.

Reply:
also, could you rename it to `docker_registry` or `extra_docker_registry` if it does not override the default registry list in the `/etc/containers`.

Reply:
hm. this modifies the existing  "docker.io" registry by adding a new mirror. This is **not** another extra registry.

Reply:
> The customized values should be controlled by yaml files in a suite. You need to raise an exception or just ignore adding registry if corresponding key is missing.

no clue where this yaml file is located.

Reply:
> hm. this modifies the existing "docker.io" registry by adding a new mirror. This is **not** another extra registry.

so maybe don't use `add` keyword in method names it does misguide

Reply:
it can be in any *suitable* under the directory `qa/suites/rados/cephadm/`